components/01-atoms/videos/_video-html5.twig

```diff
@@ -0,0 +1,19 @@
+{#
+/**
+ * Available variables:
+ * - video_object__attributes - <video> HTML5 specific attributes
+ * - video__urls - array of video URLs. Ideally, MP4, WEBM, and OGG files.
+ */
+#}
+
+{% set video_object__attributes = video_object__attributes|default({
+  'preload': '',
+  'controls': ''
+}) %}
+
+<video {{ add_attributes(video_object__attributes) }}>
+  {% for video__url in video__urls %}
+    <source src="{{ video__url }}" type="video/mp4">
+  {% endfor %}
+  Your browser does not support the video tag.
```

Comment:
@callinmullaney Could this text be made into a translatable string? `{{ "Your browser does not support the video tag"|t }}`